<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customoutdoor</class>
 <widget class="QDialog" name="customoutdoor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#customoutdoor {
  border-image:url(:/images/images/background.PNG);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#002244;</string>
     </property>
     <property name="text">
      <string>Select Outdoor Activitites (Maximum Five)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Jeep safari on Karakoram Highway amidst dramatic passes and glaciers   -----PKR 4500</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Camp , stargaze at Shandur Pass, highest polo ground  -----PKR 500</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Raft Gilgit or Indus for adrenaline &amp; mountain views    -------PKR 5000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Mountain bike Shimshal Valley's trails and meet its villagers   ------PKR 6000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Hike Naltar Valley's meadows &amp; waterfalls, ride horses, breathe fresh air    ---------PKR 1500</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Trek Fairy Meadows' wildflowers &amp; witness Nanga Parbat   ---------PKR 2000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#002244;</string>
       </property>
       <property name="text">
        <string>Rock climb Fairy Meadows for various levels and stunning views   ------PKR 1000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextbtn">
       <property name="styleSheet">
        <string notr="true">padding: 10px;
border-radius:10px;
background-color: rgb(89, 170, 163);
color:rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
